src/lib/command/smartthings-command.ts

```diff
@@ -21,37 +25,101 @@ export const smartThingsCommandBuilder = <T extends object = object>(
 			type: 'string',
 			default: 'default',
 		})
+		.option('verbose-directory-logging', {
+			describe: 'including this will log, to stderr, information about config, data, and' +
+				' logging directories found and used',
```

Comment:
What about this wording?
`log information about config, data, and logging directories to stderr`

Reply:
Heh, that's way better so I updated it. I also added a comment on the option to describe that it is temporary.